ui/pages/confirmations/confirm/confirm.tsx

```diff
@@ -1,61 +1,66 @@
 import React, { useState } from 'react';
-
-import { Page } from '../../../components/multichain/pages/page';
+import { AlertActionHandlerProvider } from '../../../components/app/alert-system/contexts/alertActionHandler';
 ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
 import { MMISignatureMismatchBanner } from '../../../components/app/mmi-signature-mismatch-banner';
 ///: END:ONLY_INCLUDE_IF
-
-import ScrollToBottom from '../components/confirm/scroll-to-bottom';
-import setCurrentConfirmation from '../hooks/setCurrentConfirmation';
-import syncConfirmPath from '../hooks/syncConfirmPath';
-
+import { Page } from '../../../components/multichain/pages/page';
+import { GasFeeContextProvider } from '../../../contexts/gasFee';
+import { TransactionModalContextProvider } from '../../../contexts/transaction-modal';
+import AdvancedGasFeePopover from '../components/advanced-gas-fee-popover';
+import { ConfirmAlerts } from '../components/confirm/confirm-alerts';
 import { Footer } from '../components/confirm/footer';
 import { Header } from '../components/confirm/header';
 import { Info } from '../components/confirm/info';
 import { LedgerInfo } from '../components/confirm/ledger-info';
 import { Nav } from '../components/confirm/nav';
 import { PluggableSection } from '../components/confirm/pluggable-section';
-import { TransactionModalContextProvider } from '../../../contexts/transaction-modal';
-import { GasFeeContextProvider } from '../../../contexts/gasFee';
-import AdvancedGasFeePopover from '../components/advanced-gas-fee-popover';
-import EditGasFeePopover from '../components/edit-gas-fee-popover';
-import { ConfirmAlerts } from '../components/confirm/confirm-alerts';
+import ScrollToBottom from '../components/confirm/scroll-to-bottom';
 import { Title } from '../components/confirm/title';
+import EditGasFeePopover from '../components/edit-gas-fee-popover';
+import setConfirmationAlerts from '../hooks/setConfirmationAlerts';
+import setCurrentConfirmation from '../hooks/setCurrentConfirmation';
+import syncConfirmPath from '../hooks/syncConfirmPath';
+import useConfirmationAlertActions from '../hooks/useConfirmationAlertActions';
 
 const Confirm = () => {
   const currentConfirmation = setCurrentConfirmation();
   syncConfirmPath();
+  setConfirmationAlerts();
+
+  const processAction = useConfirmationAlertActions();
```

Comment:
Is it possible to move 3 lines above into `AlertActionHandlerProvider` ?

Reply:
Removing this (was kept as a merge conflict resolution mistake)